wine-classification/cookiecutter.json

```diff
@@ -0,0 +1,3 @@
+{
+    "project_name": "Wine classification"
```

Comment:
good idea on this, I don't know where this gets consumed yet

Reply:
I don't know where it is used either, but I guess it is the minimal version of this: https://cookiecutter.readthedocs.io/en/stable/tutorials/tutorial1.html?highlight=cookiecutter.json#cookiecutter-json